src/server/event_bus.c

```
@@ -0,0 +1,168 @@
+void obc_event_bus_init(void)
+{
+    memset(g_events, 0, sizeof(g_events));
+    g_event_count = 0;
+}
+
+void obc_event_bus_shutdown(void)
+{
+    g_event_count = 0;
+}
+
+int obc_event_subscribe(const char *event_name, obc_event_handler_fn fn,
```

Clang (14.0.6)

[warning] parameter name 'fn' is too short, expected at least 3 characters

(readability-identifier-length)